packages/edit-site/src/components/add-new-template/utils.js

```diff
@@ -408,6 +408,62 @@ export const useTaxonomiesMenuItems = ( onClickMenuItem ) => {
 	return taxonomiesMenuItems;
 };
 
+const AUTHOR_TEMPLATE_PREFIX = { user: 'author' };
+export function useAuthorMenuItem( onClickMenuItem ) {
+	const existingTemplates = useExistingTemplates();
+	const defaultTemplateTypes = useDefaultTemplateTypes();
+	const authorInfo = useEntitiesInfo( 'root', AUTHOR_TEMPLATE_PREFIX );
```

Comment:
Here is something missing from `useEntitiesInfo` and we need to think well if we want to update(add param) or something different. We need to pass `who: 'authors'` to the query that checks if more entities exist and can return incorrect results if we have only non-author users left(subscriber for example).

Reply:
Well, that required a bigger change I included the change here, changing what useEntitiesInfo receives to allow additional query parameters we could also allow other things easily in the future.